cloudify_cli/config/parser_config.py

```diff
@@ -276,22 +279,22 @@ def parser_config():
                             },
                             '--backup-first': {
                                 'dest': 'backup_first',
-                                'help': 'Whether to backup before purging.'
-                                        'Backup will be in tar.gz format.',
+                                'help': 'Whether to backup before purging. '
+                                        'Backup will be in tar.gz format',
                                 'action': 'store_true',
                             }
                         },
-                        'help': "Delete a Manager's logs",
+                        'help': "Delete the Manager's service logs",
```

Comment:
Is there a difference between the Manager's 'logs' and its 'service logs'? Why does the 'purge' command specifically refers to 'service logs', while the two other sub commands under logs refer to 'logs'?